.github/workflows/gh-ci.yaml

```
@@ -85,7 +85,19 @@ jobs:
         install-tests: false
         installer:  mamba 
         shell: bash  -l {0} 
-      
+
+    - name: Setup Conda Environment
+      uses: mamba-org/setup-micromamba@v1
+      with:
+        environment-file: ../devtools/conda-envs/test_env.yaml
```

The path is absolute from the repo root, not this file. 

```suggestion
        environment-file: devtools/conda-envs/test_env.yaml

```